.env

```
@@ -2,7 +2,7 @@
 NEXT_PUBLIC_YORKIE_VERSION='0.7.17'
 NEXT_PUBLIC_YORKIE_JS_VERSION='0.7.17'
 NEXT_PUBLIC_YORKIE_IOS_VERSION='0.7.12'
-NEXT_PUBLIC_YORKIE_ANDROID_VERSION='0.7.12'
+NEXT_PUBLIC_YORKIE_ANDROID_VERSION='0.7.16'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)